### PR TITLE
Add setup cockpit playbook

### PR DESCRIPTION
**Pull request by HVSharma12**
Playbook to setup Cockpit as per - https://documentation.suse.com/alp/dolomite/html/alp-dolomite/available-alp-workloads.html#task-run-cockpit-with-podman

### CODE REVIEW

File: examples/ansible/setup_cockpit.yml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+      image: registry.opensuse.org/suse/alp/workloads/tumbleweed_containerfiles/suse/alp/workloads/cockpit-ws:latest
+      service:
+        name: cockpit.service
+
+  tasks:
+    - name: Retrieve image for workload {{ workload.name }}
+      containers.podman.podman_image:
+        name: "{{ workload.image }}"
+        state: present
+      register: retrieve_workload_image
```
Comment (pdostal) [inline examples/ansible/setup_cockpit.yml:24]: Not needed.

Comment (pdostal) [inline examples/ansible/setup_cockpit.yml]: Not needed.

File: examples/ansible/setup_cockpit.yml [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+- name: Setup Cockpit Web server
+  hosts: alphost
+  become: true
+  vars:
+    workload:
+      extend_functionality: 'false'
+      name: cockpit-ws
+      image: registry.opensuse.org/suse/alp/workloads/tumbleweed_containerfiles/suse/alp/workloads/cockpit-ws:latest
+      service:
+        name: cockpit.service
```
Comment (pdostal) [inline examples/ansible/setup_cockpit.yml]: The service name is static. This is not needed.

File: examples/ansible/setup_cockpit.yml [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+      ansible.builtin.debug:
+        msg: >-
+          Cockpit Web UI is running on https://{{ ansible_default_ipv4.address }}:9090
+          Please accept the warning about the self-signed SSL certificate to access it.
+
+    - name: Install cockpit-tukit for software updates functionality
+      ansible.builtin.package:
+        name: cockpit-tukit
+        state: present
+      register: package_installed
```
Comment (pdostal) [inline examples/ansible/setup_cockpit.yml]: This is not needed.

**Review by pdostal — APPROVED** [review]
The code looks OK to me now :rocket:

**Review by brett060102 — APPROVED** [review]
LGTM

**Comment by pdostal** [comment]
Will there be other people reviewing? This is public repository after all.

**Comment by brett060102** [comment]
I don't know for sure, but I doubt it. The project does not get a lot of traffic and Fergal (rtamalin) is tied up on other work.

**Pull request merged**